### PR TITLE
Add Play : Play Fair Cipher

### PR DESCRIPTION
**Pull request by KMohZaid**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New Play

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 
1. Local Testing
2. E2E testing
# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] ~~I have made corresponding changes to the documentation~~
- [x] My changes generate no new warnings
- [ ] ~~I have added tests that prove my fix is effective or that my feature works~~
- [x] New and existing unit tests pass locally with my changes
- [ ] ~~Any dependent changes have been merged and published in downstream modules~~

### CODE REVIEW

**Comment by KMohZaid** [comment]
Why my email is showing instead username?
@atapas

**Review by bhtibrewal — COMMENTED** [review]
Screenshot 2022-10-19 at 11 30 36 PM
Screenshot 2022-10-19 at 11 31 37 PM
I am not getting the same input text after encrypting and then decrypting

**Comment by KMohZaid** [comment]
> Screenshot 2022-10-19 at 11 30 36 PM
> 
> Screenshot 2022-10-19 at 11 31 37 PM
> 
> I am not getting the same input text after encrypting and then decrypting

It is because your input text is odd lenght so while encrypting x is added to make it even

And yeah if your input contain same character pair like hello = he ll o then it will split ll using x also
So it is he lx lo and then encrypt it


This is how play fair works 

More info on play fair cipher:
https://www.dcode.fr/playfair-cipher
https://www.geeksforgeeks.org/playfair-cipher-with-examples/

**Comment by koustov** [comment]
@KMohZaid Can you check your tab settings in your editor? It is too large to review

**Comment by KMohZaid** [comment]
> @KMohZaid Can you check your tab settings in your editor? It is too large to review

What you mean by Tab settings?
Too large? Isn't it only 2 file ( css and .tsx ) other are just merge from this repo

**Comment by koustov** [comment]
> > @KMohZaid Can you check your tab settings in your editor? It is too large to review
> 
> What you mean by Tab settings? Too large? Isn't it only 2 file ( css and .tsx ) other are just merge from this repo

I meant the edit tab size settings. e.g.: In my vs code its
image

Check yours

**Comment by KMohZaid** [comment]
> I meant the editor tab size settings. e.g.: In my vs code its image
> 
> Check yours

Mine is 4 but prettier format javascript to 2

**Comment by atapas** [comment]
> Why my email is showing instead username?
> 
> @atapas 

Go to your GitHub profile, edit it, add your full name. It will be fixed automatically here.

**Comment by KMohZaid** [comment]
> > Why my email is showing instead username?
> > @atapas
> 
> Go to your GitHub profile, edit it, add your full name. It will be fixed automatically here.

Tried but not happen ig it is because of nhost  because there also my email is written in place of name

File: src/plays/play-fair-cipher/styles.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+}
+.pf-form > label {
+	font-size: 1.125rem;
+	font-weight: 700;
+	line-height: 1.75rem;
+}
+.pf-form > input[type="text"] {
+	--tw-border-opacity: 1;
+	--tw-text-opacity: 1;
+	border-color: rgb(156 163 175 / var(--tw-border-opacity));
```
Comment (ammaaraslam) [inline src/plays/play-fair-cipher/styles.css:23]: Is a TW variable necessary in here? Can't you use a regular value

**Review by ammaaraslam — CHANGES_REQUESTED** [review]

File: src/plays/play-fair-cipher/styles.css (hunk 1 above)
Comment (KMohZaid) [reply src/plays/play-fair-cipher/styles.css:23]: I just use tailwind generate css

**Review by atapas — APPROVED** [review]

**Comment by KMohZaid** [comment]
> > > Why my email is showing instead username?
> > > @atapas
> > 
> > 
> > Go to your GitHub profile, edit it, add your full name. It will be fixed automatically here.
> 
> Tried but not happen ig it is because of nhost because there also my email is written in place of name

@atapas any solution?
I edit full name but still email showing

**Pull request merged**